src/instructlab/configuration.py

```diff
@@ -627,10 +631,14 @@ def init(
             config_obj.general.log_level = "DEBUG"
             config_obj.general.debug_level = debug_level
 
+        if log_format != log.FORMAT:
+            config_obj.general.log_format = log_format
+
         # setup logging
         log.configure_logging(
             log_level=config_obj.general.log_level.upper(),
             debug_level=config_obj.general.debug_level,
+            fmt=log_format,
```

Comment:
```suggestion
            fmt=config_obj.general.log_format,
```

Like the prior line, shouldn't this come from the config object?

Reply:
I changed the code, PTAL again, first we check for the flag, then try the config, then the default. Thanks